__tests__/api.test.js

```diff
@@ -153,3 +154,40 @@ describe(" GET /api/articles/:article_id/comments", () =>{
         })
     })
 })
+
+describe("POST:/api/articles/:article_id/comments", ()=>{
+    test("POST:posted comment with username and body property", ()=>{
+        const newComm={author:'icellusedkars', body:" I carry a log — yes. Is it funny to you? It is not to me."}
+        return request (app)
+        .post ('/api/articles/1/comments')
+        .send(newComm)
+        .expect(201)
+        .then((response) =>{
+            const comments=response.body.comments
+            expect(comments.author).toBe('icellusedkars')
+            expect(comments.body).toBe(" I carry a log — yes. Is it funny to you? It is not to me.")
+        })
+    })
+    test("POST:posted comment with not writing username ", ()=>{
+        const newComm={author:' ', body:'I carry a log — yes. Is it funny to you? It is not to me.'}
+        return request (app)
+        .post ('/api/articles/1/comments')
+        .send(newComm)
+        .expect(400)
+        .then((response) =>{
+            expect(response.body.msg).toBe('Bad Request:username is missing')
+            
+        })
+    })
+    test("POST:posted comment wihtout a body ", ()=>{
+        const newComm={author:'icellusedkars ', body:' '}
```

Comment:
when test using an empty string dont put spaces because once you put a space it doesn't count as an empty string . so instead of " " use ""